src/databases/WavefrontOBJ/WavefrontOBJPluginInfo.C

```diff
@@ -113,13 +113,18 @@ WavefrontOBJGeneralPluginInfo::HasWriter() const
 //
 //  Programmer:  generated by xml2info
 //  Creation:    omitted
+// 
+//  Modifications:
+//    Justin Privitera, Tue Nov 28 17:31:40 PST 2023
+//    Added "*.mtl".
 //
 // ****************************************************************************
 std::vector<std::string>
 WavefrontOBJGeneralPluginInfo::GetDefaultFilePatterns() const
 {
     std::vector<std::string> defaultPatterns;
     defaultPatterns.push_back("*.obj");
+    defaultPatterns.push_back("*.mtl");
```

Comment:
Do we need to specify `.mtl` files here? 

For *reading*, a user will never open a `.mtl` file directly, will they? An `.mtl` file is just an optional companion file to an `.obj` file.

Is there a reason we need to specify this for the writer?

Reply:
You know, I just threw that in there thinking "huh, maybe I forgot about this last time; looks important." I don't know if I understood fully what that was for. I can remove that.

Reply:
I don't think it should be there. That list should include only file extensions for files a user will either click on in the GUI or include in a CLI `OpenDatabase()` call.

There can be other, companion, files involved but since those are not the open *entry point* for VisIt to identify a plugin and open the "file", they are not included in the list. Another example is Xdmf format. The extension is `.xmf` even though there are hdf5 companion files, those are not listed in the default file patterns.